src/api/users/routes.js

```diff
@@ -40,10 +43,16 @@ const deleteUserProject = require("./controllers/deleteUserProject");
 
 router.get("/", superadmin(), getAll);
 router.get("/:id", user(), getOne);
+
 router.post("/", superadmin(), bodyValidator(userSchema), post);
+router.post("/:userId/projects/:projectId", superadmin(), createUserProject);
```

Comment:
Only a superadmin can link an user with a project ? Should be admin no ?

Reply:
@MatthiasWanner @Aeryle can help maybe ??? :)